src/io/aleph/dirigiste/Pool.java

```diff
@@ -320,12 +330,11 @@ private void startControlLoop(int duration, int iterations) {
                 }
```

Comment:
If it `break`s here, lock won't be released

Reply:
3dbbde7b5908017107d28668be065dc742f910a1

Reply:
No longer relevant.